src/Core/Users/Userlogin.php

```diff
@@ -0,0 +1,158 @@
+<?php
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+interface Userlogin implement UserInterface
```

Comment:
### Issues
- Interface can **NEVER** implement another interface. They can extend. Classes implement
- Its `implements` **NOT** `implement`.
- You implement interfaces and extend classes